Libplanet.Net/IceServer.cs

```diff
@@ -10,24 +9,26 @@ namespace Libplanet.Net
     public class IceServer
     {
         public IceServer(
-            IEnumerable<string> urls,
+            string url,
             string? username = null,
             string? credential = null)
-            : this(urls.Select(u => new Uri(u)), username, credential)
         {
+            Url = new Uri(url);
+            Username = username;
+            Credential = credential;
         }
```

Comment:
Any reason redirection to `IceServer(Uri, string?, string?)` was removed? Redirection seems more DRY. 😗

Reply:
You mean this code is repetitive with the codes between line [21-29](https://github.com/Akamig/libplanet/blob/b863d36fcb100fb782b1f7b1ccb8dc9cb3c5f02f/Libplanet.Net/IceServer.cs#L21)?
I tried
```       
public IceServer(
            string url,
            string? username = null,
            string? credential = null)
            : this(url, username, credential)
        {
        }
```
But it didn't worked. how can I redirect string type url initializer to that Initializer?

Reply:
You'd want something like

```csharp
public IceServer(
            string url,
            string? username = null,
            string? credential = null)
            : this(new Uri(url), username, credential)
        {
        }
```

Otherwise, you are recursively referencing the same constructor as there is no signature change. 🙃

Reply:
Yeap, this is what I wanted.